src/main/java/org/humancellatlas/ingest/biomaterial/web/BiomaterialController.java

```diff
@@ -69,6 +70,7 @@ public class BiomaterialController {
     private @Autowired
     MetadataLinkingService metadataLinkingService;
 
+    @PreAuthorize("#submissionEnvelope.editable")
```

Comment:
I'm not very familiar with this annotation, how does this work? why is it being called as `submissionEnvelope.editable` instead of `submissionEnvelope.isEditable()` ?

Reply:
ah i didn't know that's possible, ty for explaining!

Reply:
@aaclan-ebi Actually, the error message you get isn't all that useful. It's just a 500 so I will try to customize it

Reply:
I would guess that we'd want to return HTTP 403 Forbidden?